data/card_catalogue.json

```diff
@@ -748,177 +790,97 @@
         {
           "title": "Reload",
           "upgrade": "Tactical Reload",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Return a Card with Blue Rarity from your discard pile to your hand",
```

Comment:
We don't have a discard pile. I think people understand this without that addition